src/map.rs

```diff
@@ -1307,25 +1294,25 @@ where
         // there are three cases when a bin pointer is invalidated:
         //
         //  1. if the table was resized, bin is a move entry, and the resize has completed. in
-        //     that case, the table (and all its heads) will be dropped in the next epoch
-        //     following that.
+        //     that case, the table (and all its heads) will be retired.
```

Comment:
These changes aren't _quite_ right. This paragraph is reasoning about when in the future the bin pointer will actually be _reclaimed_. I _think_ it should read something like:

1. if the table was resized, bin is a move entry, and the resize has completed. in that case, the table (and all its heads) has already been retired.